### ISSUE TITLE
Error in output in single line

### ISSUE THREAD

**Issue by pwsandoval** [astral-sh/ruff#12658]
It's a very quick thing and maybe solved with a flag that already exists but you didn't find it. Today I updated ruff to the latest version and I see that now the check command is more verbose and gives more details than previous versions:

Now:

![ruff_now](https://github.com/user-attachments/assets/8e6147e2-32bc-48a5-a4ae-f8ca1ad9bd4f)

Before (from the internet) because I didn't take a screen before:

<img width="910" alt="ruff_before" src="https://github.com/user-attachments/assets/05550db9-2e30-4419-bf48-0f49abe1e40c">

Basically it prints one line for each error, instead of 4 lines spelling out the position with ^^^^^.

**Comment by ember91** [comment]
This is intended and was introduced with ruff 0.5.0. See https://astral.sh/blog/ruff-v0.5.0

Pass `-output-format=concise` for the previous behavior.

**Comment by mmarras** [comment]
Is there an option to set the output-fomat flag in the pyproject.toml too?

**Comment by mmarras** [comment]
> Is there an option to set the output-fomat flag in the pyproject.toml too?

Found it:

```toml
[tool.ruff]
output-format = 'concise'
```
